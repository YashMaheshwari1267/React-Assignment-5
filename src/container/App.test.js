import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('rendering components',() => {
  it('should render the <App /> component',() => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  })

})