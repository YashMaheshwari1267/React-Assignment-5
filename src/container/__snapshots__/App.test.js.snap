// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component should render the <App /> component 1`] = `ShallowWrapper {}`;

exports[`rendering components should render the <App /> component 1`] = `ShallowWrapper {}`;
